src/store.js

```diff
@@ -11,33 +11,72 @@ let axiosClient = axios.create({
 export default new Vuex.Store({
   state: {
     list: [],
-    selectedIssue: null
+    selectedIssue: null,
+    comments: [],
+    firstPage: 1,
```

Comment:
why this property is called `firstPage`? What if you setup there 2 or 3 selecting another page?

Reply:
Changed the name from "firstPage" to "firstPaginationButton".